source/_docs/guides/drupal-8-simplesamlphp.md

```diff
@@ -0,0 +1,264 @@
+---
+title: How to Implement SimpleSAMLphp as idP for Drupal 8 on Pantheon 
+description: Setting up an identity provider on Drupal 8 using SimpleSAMLphp.
+draft: true
+tags: [code, drupal-8]
+categories: [drupal, developing]
+contributors: Javi-er
+---
+
+There are a number of articles detailing how to implement the SAML 2.0 protocol, but if you want to host your site on Pantheon and use Drupal 8 as your identity provider, there are a few details to take in account.
+<div class="alert alert-info">
+<h4 class="info">Note</h4><p markdown="1">The following tutorial assumes that you are on a linux based computer or a Mac. Also that your project is running on Apache.
+</p>
+</div>
+
+## Before You Begin
+
+Before starting with this guide, make sure you have the following:
+
+ - Have a working Drupal 8 site.
+ - Download the latest version of [SimpleSAMLphp](https://simplesamlphp.org/download).
+ - Download the Drupal module [saml_idp](https://github.com/Javi-er/saml_idp) (you’ll notice that this repo is a fork of [this one](https://github.com/bradjones1/saml_idp) this is because there were some changes necessary to make it work which are [described here](https://www.drupal.org/project/saml_idp/issues/3013357).)
+ 
+## Getting started
+
+Your project structure and configuration needs to be tweaked so it's recognized by Pantheon when deployed to the server.
+
+### Setting up SAML Idp module
+By now you should have SAML Idp module on your Drupal 8 site, go to the admin interface and enable it now.
+
+After this create the subdirectory ``/simplesamlphp/drupalauth`` and create an empty file with the name `default_enable` in that directory in order to enable the Drupal 8 integration. You can also do this with Drush executing
+
+``` bash
+drush ev 'Drupal\saml_idp\Install::postInstall()'
+```
+
+### Setting up SimpleSAMLphp
+
+1.  Place your Drupal 8 projected inside a ``web`` [nested docroot](https://pantheon.io/docs/nested-docroot).
```

Comment:
Why is nested docroot required?

Reply:
Now that you mentions it I don't think it's required, I assumed it was because simplesaml needs to be installed inside "/private" which I'm not sure if it can live in the same level as the rest of the web files? I removed the reference to nested docroot anyway.